ribbon/src/main/java/feign/ribbon/LoadBalancingTarget.java

```diff
@@ -79,7 +94,7 @@ public String name() {
 
   @Override
   public String url() {
-    return name;
+    return String.format("%s://%s", scheme, path);
```

Comment:
Just found in my own testing this isn't just scheme + path, needed the "://". :P

Thanks!